src/source/CommandLine/Microsoft.DotNet.Cli.CommandLine.csproj

```diff
@@ -11,6 +11,8 @@
 
   <PropertyGroup>
     <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <DebugType>full</DebugType>
```

Comment:
I don't think we want Windows PDBs. This shouldn't be necessary. The CI build produces standalone Portable PDBs.